src/main/java/pro/sky/calculator/controller/CalculatorController.java

```diff
@@ -0,0 +1,60 @@
+package pro.sky.calculator.controller;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import pro.sky.calculator.service.CalculatorService;
+
+import static java.lang.Integer.*;
+
+@RestController
+@RequestMapping("calculator")
+
+public class CalculatorController {
+    private CalculatorService calculatorService;
+
+    public CalculatorController(CalculatorService calculatorService) {
+        this.calculatorService = calculatorService;
+    }
+
+    @GetMapping
+    public String hello() {
+        return "Добро пожаловать в калькулятор";
+    }
+
+    @GetMapping("plus")
+    public String plus(@RequestParam(name = "num1", required = false) Integer num1, @RequestParam(name = "num2", required = false) Integer num2) {
+        if (num1 == null || num2 == null) {
+            return "Введите число";
+        }
+        return num1 + " + " + num2 + " = " + calculatorService.plus(num1, num2);
+    }
+
+    @GetMapping("minus")
+    public String minus(@RequestParam(name = "num1", required = false) Integer num1, @RequestParam(name = "num2", required = false) Integer num2) {
+        if (num1 == null || num2 == null) {
+            return "Введите число";
+        }
+        return num1 + " - " + num2 + " = " + calculatorService.minus(num1, num2);
+    }
+
+    @GetMapping("multiply")
+    public String multiply(@RequestParam(name = "num1", required = false) Integer num1, @RequestParam(name = "num2", required = false) Integer num2) {
+        if (num1 == null || num2 == null) {
+            return "Введите число";
+        }
+        return num1 + " * " + num2 + " = " + calculatorService.multiply(num1, num2);
+    }
+
+    @GetMapping("divide")
+    public String divide(@RequestParam(name = "num1", required = false) Integer num1, @RequestParam(name = "num2", required = false) Integer num2) {
+        if (num1 == null || num2 == null) {
+            return "Введите число";
+        }
+        if (num1 == 0 || num2 == 0) {
```

Comment:
```suggestion
        if (num2 == 0) {
```
числитель можешь быть равен 0